### ISSUE TITLE
Conflicting golangci-lint config file settings for `unstable` image

### ISSUE THREAD

**Issue by atc0005** [atc0005/go-ci#583]
From a CI run for a dependent project:

```
level=info msg="[config_reader] Config search paths: [./ /__w/golang-oracle/golang-oracle /__w/golang-oracle /__w / /github/home]"
level=info msg="[config_reader] Used config file ../../../.golangci.yml"
level=error msg="Running error: linter \"stylecheck\" can't be disabled and enabled at one moment"
level=info msg="Memory: 2 samples, avg is 44.0MB, max is 44.0MB"
level=info msg="Execution took 4.704396ms"
```

**Comment by atc0005** [comment]
See also #574.

**Comment by atc0005** [comment]
Fixed via #586.
